apps/fileman/ChangeLog

```diff
@@ -1,3 +1,4 @@
 0.01: New app!
 0.02: Improve handling of large amounts of files (fix #579)
 0.03: Update RegExp use (Was using backreference instead of character code)
+0.04: Allow managing files in the app loader
```

Comment:
There is no need to increment the version number

Reply:
Ok, I removed it. It's true that there's nothing to update on the watch, but how are people going to know about this new feature in the app loader?

Reply:
Same kind of way - they'd find new versions of the app in the app loader and they'll come across the interface changes there, although you are right in a way, in that they'd have to find the app in the app loader to start with.

Perhaps people would find it useful if the app loader summarised available `interface.html`s for their installed apps?